src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -3,6 +3,7 @@
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 
 public class ShooterConstants {
+    public static final double DEFAULT_TARGET_VELOCITY = 2500;
```

Comment:
Would "default" mainly be used when next to the tower?

Reply:
Default is when the limelight can't see the hub

Reply:
Changle name to IdleVelocity